### PR TITLE
transferring r-harmony from Bioconda

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mfansler** [comment]
I'm fine with accepting this as a transfer, but please use [`conda_r_skeleton_helper`](https://github.com/bgruening/conda_r_skeleton_helper) to generate the recipe.

File: recipes/r-harmony/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    - r-tibble
+
+test:
+  commands:
+    - $R -e "library('harmony')"           # [not win]
+    - "\"%R%\" -e \"library('harmony')\""  # [win]
+
+about:
+  home: software.broadinstitute.org/harmony
+  license: GPL-3.0-or-later
```
Comment (mfansler) [inline recipes/r-harmony/meta.yaml:68]: Link is dead; `GPL-3` in DESCRIPTION entails `GPL-3.0-only`:

```suggestion
  home: https://portals.broadinstitute.org/harmony/
  dev_url: https://github.com/immunogenomics/harmony
  license: GPL-3.0-only
```

File: recipes/r-harmony/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    - {{ cran_mirror }}/src/contrib/Archive/harmony/harmony_{{ version }}.tar.gz
+  sha256: a63c7d7cbbc5d183e8f919552e9d73044e0a89660856e80861a00eb5d25ac7b5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```
Comment (mfansler) [inline recipes/r-harmony/meta.yaml:21]: Windows will need:

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'            # [win]
    - '*/Rblas.dll'        # [win]
    - '*/Rlapack.dll'      # [win]

```

**Review by mfansler — CHANGES_REQUESTED** [review]
Some adjustments to make.

File: recipes/r-harmony/meta.yaml (hunk 2 above)
Comment (mfansler) [reply recipes/r-harmony/meta.yaml:21]: FYI, here's a reference for this: https://github.com/conda/conda-build/pull/4786

File: recipes/r-harmony/meta.yaml (hunk 1 above)
Comment (dillonroach) [reply recipes/r-harmony/meta.yaml:68]: Thanks much for the extra help

File: recipes/r-harmony/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    - r-cowplot
+    - r-dplyr
+    - r-ggplot2
+    - r-rlang
+    - r-tibble
+
+test:
+  commands:
+    - $R -e "library('harmony')"           # [not win]
+    - "\"%R%\" -e \"library('harmony')\""  # [win]
```
[the remark this answers is not in the document]
Comment (dillonroach) [reply recipes/r-harmony/meta.yaml:69]: Great addition - thanks, I'll be sure it gets included with the win64 build fix

File: recipes/r-harmony/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    - r-rcppprogress
+    - r-rhpcblasctl
+    - r-cowplot
+    - r-dplyr
+    - r-ggplot2
+    - r-rlang
+    - r-tibble
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
```
Comment (mfansler) [inline recipes/r-harmony/meta.yaml:54]: win fix:

```suggestion
    - {{ native }}gcc-libs            # [win]
    - {{ native  }}libwinpthread-git  # [win]
```

File: recipes/r-harmony/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    - '*/Rblas.dll'        # [win]
+    - '*/Rlapack.dll'      # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('m2w64_c') }}    # [win]
```
Comment (mfansler) [inline recipes/r-harmony/meta.yaml:32]: remove duplicated requirement

```suggestion
```

**Comment by dillonroach** [comment]
@mfansler thanks for the help on this one; greatly appreciated!

**Comment by mfansler** [comment]
@dillonroach please [create a PR on Bioconda](https://github.com/bioconda/bioconda-recipes/pulls ) to remove the recipe. I'll merge this once that is linked here.

**Comment by dillonroach** [comment]
https://github.com/bioconda/bioconda-recipes/pull/45982 - @mfansler PR up, I'm happy to wait for that to go through

**Comment by johanneskoester** [comment]
Pr on bioconda is merged

**Comment by mfansler** [comment]
Thanks for taking the time to work on this! ❤️

**Pull request merged**